### DISCUSSION TITLE
Self hosted setup

### DISCUSSION THREAD

**Discussion by luwol03** [uxbox/uxbox#1781] [Q&A] [answered]
What is the desired way to set up penpot without disabling the secure cookies? To do that there must be a way to tell penpot use this certificates. Even if I use a Reverse Proxy Right, because there is one way without https from the server to the Reverse Proxy?

**Comment by niwinz** [comment, marked as answer]
secure cookies are not the problem for local, private installations at all, this is just a mark for the browser for enable sending cookies on non HTTPS environment.

Un summary: use https, or just don't use "secure" flag on cookies (both are ok for the different case uses)

**Reply by luwol03** [reply]
Thank you for clarification.
